tests/test_environment.py

```diff
@@ -101,6 +101,66 @@ def test_ignore(client):
     )
 
 
+@pytest.mark.django_project(
+    extra_settings="""
+    TEMPLATE_LOADERS = (
+        'django.template.loaders.filesystem.Loader',
+        'django.template.loaders.app_directories.Loader',
+    )
+    ROOT_URLCONF = 'tpkg.app.urls'
+    """
+)
+def test_invalid_template_variable_marker_cleanup(django_testdir) -> None:
+    django_testdir.create_app_file(
+        """
+        from django.urls import path
```

Comment:
I think it will be easier to have the tests render the template directly using e.g. [`render_to_string`](https://docs.djangoproject.com/en/4.2/topics/templates/#django.template.loader.render_to_string) instead of setting up views and requests and everything.

Reply:
Test updated